src/Server.ts

```diff
@@ -10,6 +11,7 @@ class Server{
   }
 
   public bootstrap: any = () => {
```

Comment:
bootstrap should return Server because it return this and this refer to the class (Server)